aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -993,4 +994,93 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
       onesidedOpt, lengthOpt, /*return_complex=*/false);
 }
 
+void _fft_fill_with_conjugate_symmetry_(const Tensor& input, IntArrayRef dim_) {
+  const auto input_sizes = input.sizes();
+  const auto input_strides = input.strides();
+  TORCH_CHECK(dim_.size() > 0);
+  DimVector dim(dim_.begin(), dim_.end());
+  at::maybe_wrap_dims(dim, input_strides.size());
+
+  if (input.numel() == 0 || input_sizes[dim.back()] <= 2) {
+    return;  // No elements need writing
+  }
+
+  // Small dimensions may be treated as batch dims since they don't get mirrored
+  dim.erase(
+      std::remove_if(dim.begin(), dim.end(), [&](int64_t dim) {
+        return (input_sizes[dim] <= 2);
+      }),
+      dim.end());
+
+  // Use TensorIterator to coalesce batch dimensions
```

Comment:
I feel like this section, and the code through 1048, needs more commentary to explain what's going on

Reply:
I've included more comments, let me know if it's still unclear.